server/tweetSearch.js

```diff
@@ -4,6 +4,9 @@ module.exports = function(client) {
     var hashtags = ["#bristech", "#bristech2016"];
 
     function addTweetItem(tweets, tag) {
+        if (tweets.length === 0) {
+            return;
```

Comment:
The function as a whole doesn't return anything, it's just used to add to the tweet store